### DISCUSSION TITLE
Offline Lyrics

### DISCUSSION THREAD

**Discussion by PatAbah** [gokadzev/Musify#140] [Ideas]
Any way of getting music lyrics to work offline after downloading?

The lyrics feature on this app is a really nice feature, however, I noticed that it doesn't work for local tracks (even local tracks downloaded by Musify). Lyrics only work while streaming.

I think it'll be nice to store the lyrics file in a directory (with an option to Download lyrics for tracks who's lyrics file weren't found in the lyrics directory —including offline tracks). 

Any hope of getting offline Lyrics soon?

Thanks!

**Comment by gokadzev** [comment]
> Any way of getting music lyrics to work offline after downloading?
> 
> The lyrics feature on this app is a really nice feature, however, I noticed that it doesn't work for local tracks (even local tracks downloaded by Musify). Lyrics only work while streaming.
> 
> I think it'll be nice to store the lyrics file in a directory (with an option to Download lyrics for tracks who's lyrics file weren't found in the lyrics directory —including offline tracks).
> 
> Any hope of getting offline Lyrics soon?
> 
> Thanks!

Hello, do you know what type of file can we use for lyrics? (I'm thinking about JSON only)

**Reply by PatAbah** [reply]
I'm glad you're actually considering this.

Well, I've come across a couple of lyrics services. Some used XML, however, majority of them used JSON only. I think json is great!
